resources/public/api/conf/1.0/retrieve_income_tax_crystallisation.yaml

```diff
@@ -1,5 +1,5 @@
 get:
-  summary: Retrieve Income Tax (Self Assessment) Crystallisation Obligations
+  summary: Retrieve Income Tax (Self Assessment) Final Declaration Obligations
```

Comment:
Name on spec is `Retrieve Income Tax (Self Assessment) Crystallisation Obligations`